src/RobotInterface/YarpImplementation/include/BipedalLocomotion/RobotInterface/YarpSensorBridgeImpl.h

```diff
@@ -608,8 +525,13 @@ struct YarpSensorBridge::Impl
      *  Get number of MAS Sensors
      */
     template <typename MASSensorType>
-    std::size_t getNumberOfMASSensors(MASSensorType* sensorInterface)
+    std::size_t getNumberOfMASSensors(MASSensorType*& sensorInterface) 
     {
+        if (sensorInterface == nullptr)
+        { 
+            return -1;
```

Comment:
`size_t` is an unsigned integer. I'm afraid that this will return the maximum number

Reply:
Thank you! Fixed in https://github.com/dic-iit/bipedal-locomotion-framework/pull/182/commits/d4dcb8a28a7f4b4a59fee3f7b96e2f25f6a4d268.